recipes/python-can/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "python-can" %}
+{% set version = "2.1.0" %}
+{% set sha256 = "4a5c01dd67feeda35f88e6c12ea14ac8cabd426b9be0cc5f9fd083fe90a9dbfc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - pyserial
+    - setuptools
+
+test:
+  imports:
+    - can
+
+about:
+  home: https://github.com/hardbyte/python-can
+  license: LGPLv3+
+  license_family: LGPL
+  license_file: LICENSE.txt
+  summary: 'The can package provides controller area network support for Python developers '
+  description: |
+    The **C**\ ontroller **A**\ rea **N**\ etwork is a bus standard designed
+    to allow microcontrollers and devices to communicate with each other. It
+    has priority based bus arbitration, reliable deterministic
+    communication. It is used in cars, trucks, boats, wheelchairs and more.
+    The ``can`` package provides controller area network support for
+    Python developers; providing `common abstractions to
+    different hardware devices`, and a suite of utilities for sending and receiving
+    messages on a can bus.
+    The library supports Python 2.7, Python 3.4+ as well as PyPy 2 & 3 and runs on Mac, Linux and Windows.
+
+  doc_url: https://python-can.readthedocs.org/en/stable/
+  dev_url: https://github.com/hardbyte/python-can
+
+extra:
+  recipe-maintainers:
+    - hardbyte
```

Comment:
@hardbyte are you interested in being a maintainer?

Reply:
Sure